manifests/art.yaml

```diff
@@ -1,5 +1,5 @@
 updates:
-  - file: "{MAJOR}.{MINOR}/cluster-kube-descheduler-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
+  - file: "stable/cluster-kube-descheduler-operator.v{MAJOR}.{MINOR}.0.clusterserviceversion.yaml" # relative to this file
```

Comment:
Is renaming the `{MAJOR}.{MINOR}` folder into `stable` necessary? I see the benefit of reducing the number of changes when bumping the version. Nevertheless, is this a pattern every OLM managed operator is expected to follow?

Reply:
According https://issues.redhat.com/browse/ART-3107, all operators should adopt the `stable` pattern